### ISSUE TITLE
Smooth scrolling issue with certain modules

### ISSUE THREAD

**Issue by tuomas2** [AndBible/and-bible#1710]
>  I usually run split screen King James on top NIV on botton but it Does not scroll smooth more when running split screen. The text blinks on and off on top half and sometimes just goes black.  The menu bottons in the top do not show . They are there but they don't show just a black bar on the top I can guess where they are and they work but it's just a guess because I can't see them. I did a back up and tried uninstalling and loading again new. 

https://support.andbible.org/scp/tickets.php?id=746

**Comment by cmahte** [comment]
Is this 2nd work mentioned sideloaded?  It might be better to remove the name and focus on where it came from, and how to deal with imperfect side loaded works (attempt to enforce code strictness or do basic checks and reject them?)

**Comment by cmahte** [comment]
That is eventually something like this will start showing up in google searches, and cause confusion and questions about where it came from.

**Comment by tuomas2** [comment]
If I remember correctly NIV has once been publicly available via Crosswire, similarly as ESV was. 

Users are educated via Downloads > NIV "fake" why module is not available.

**Comment by tuomas2** [comment]
I would like to take a look at the OSIS fragments in that particular NIV module, and see what is the root cause for this issue. It might affect many modules that we are not aware of. But someone need to provide me that first as I don't have it...

**Comment by cmahte** [comment]
As far as I know, Crosswire never hosted NIV (and I've been an active user
of Sword since about 1998).

Most of the time, when someone says they have an NIV, they made it
themselves.

There may have been 2 possible "someone else made it" releases:

Karl Kleinpaste left an NIV visible on a private server at kleinpaste.org
for a short time.  It wasn't in the sword tree, but on the same server.  If
was only visible if you logged directly into ftp://kleinpaste.org
(something like kleinpaste.org/private/sword/mods... ) ... It was labeled
private, but left available to anonymous logins.

There may have also been an NIV visible for a short time from ebible.org
due to a similar misconfiguration. But I'm less certain if Michael ever had
permission to release the NIV in any form.

Both of these required basically hacking the site to access. That is,
manually logging in FTP and browsing to even see the module, then
downloading and sideloading the module. And both of these were many years
ago (maybe 2009-10 Kleinpaste and maybe 2011-14 for ebible). and the number
of actual downloads on them was likely in the 10s, not even 100s. If either
of these are the source, they weren't directly loaded onto an Android
device that still works. Anyone with an NIV module in a currently used
android instance of Andbible sideloaded it, unless there are repositories
out there I'm not aware of.

Also, at one point, there was a semi-automatic tool that one could use to
webget an entire bible from a specific website and it compiled a sword
module from that. This tool created sort of broken modules.  The exposed
module on Karl's website looked exactly or very similar to the output of
this tool. you might find traces of this if you search the sword developer
discussion list.  It was produced by an enthusiast who was reprimanded and
basically excommunicated for his work. This was even older  time frame
(before 2010, maybe even before 2000)

**Comment by tuomas2** [comment]
Changed title

**Comment by tuomas2** [comment]
Thanks for good input!
